lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>"v základní instalaci" -&gt; "vestavěný"? To o JavaScriptu bych asi klidně vypustil, většina holek o JS ani nic neví.</p><p> </p><p>Přirovnání ke slovníku je myslím super. Klidně bych to rozvedl a řekl něco jako že JSON odpoídí slovníku, ve kterém jsou další slovníky / listy / inty / floaty / booly nebo Nony. A ani bych neváhal vysvětlit co to je (de)serializace :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Nechal jsem oboje :) základní instalaci i vestavěný. JS odmazáno